README.md

```diff
@@ -150,30 +155,30 @@ Add an inline policy to the IAM user or EC2 instance role to allow it to use EC2
 }
 ```
 
-#### 4. Create EC2 Fleet / Auto-Scaling Group
-
-*Create EC2 Spot Fleet*
+#### 4. Create an Auto Scaling Group or Spot Fleet
 
-https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-fleet-requests.html#create-spot-fleet
+https://docs.aws.amazon.com/cli/latest/reference/autoscaling/create-auto-scaling-group.html
+Here is a [getting started tutorial for ASG](https://docs.aws.amazon.com/autoscaling/ec2/userguide/GettingStartedTutorial.html).
 
 Make sure that you:
-- Check ```Maintain target capacity``` [why](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-fleet-configuration-strategies.html#ec2-fleet-request-type)
 - Specify an SSH key that will be used later by Jenkins.
+- Follow [Spot best practices](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/spot-best-practices.html), if using Spot.
 
-*Alternatively, create an Auto Scaling Group*
-
-https://docs.aws.amazon.com/autoscaling/ec2/userguide/GettingStartedTutorial.html
+> [!WARNING]  
+> AWS strongly discourages using the SpotFleets because it is now categorized as legacy API with no planned investment.
+Use Auto Scaling Groups instead.
+> [Spot Fleet documentation](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_RequestSpotFleet.html)
 
 #### 5. Configure Jenkins
 
-Once the Spot Fleet or ASG is ready, you can use it by adding a new **EC2 Fleet** cloud in the Jenkins configuration.
+Once your ASG or Spot Fleet is ready, you can use it by adding a new **EC2 Fleet** cloud in the Jenkins configuration.
 
 1. Goto ```Manage Jenkins > Plugin Manager```
 1. Install ```EC2 Fleet Jenkins Plugin```
-1. Goto ```Manage Jenkins > Configure System```
+1. Goto ```Manage Jenkins > Configure Clouds```
```

Comment:
This workflow is also a little different for the newer Jenkins versions. Ignore if same as above.